docs/csharp/language-reference/tokens/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "C# Tokens (C# Reference) | Microsoft Docs"
+ms.date: "2017-02-14"
+ms.prod: .net
+ms.technology: 
+  - "devlang-csharp"
+ms.topic: "article"
+f1_keywords: 
+  - "cs.tokens"
+dev_langs: 
+  - "CSharp"
+helpviewer_keywords: 
+  - "tokens [C#]"
+  - "C# language, tokens"
+  - "Visual C#, tokens"
+  - "@ token (C#)"
+  - "@ token (C#)"
+ms.assetid: 4c5c0539-2e37-40b7-91ce-75af5aabd3f9
+caps.latest.revision: 22
+author: "rpetrusha"
+ms.author: "ronpet"
+translation.priority.ht: 
+  - "cs-cz"
+  - "de-de"
+  - "es-es"
+  - "fr-fr"
+  - "it-it"
+  - "ja-jp"
+  - "ko-kr"
+  - "pl-pl"
+  - "pt-br"
+  - "ru-ru"
+  - "tr-tr"
+  - "zh-cn"
+  - "zh-tw"
+---
+
+# C# Tokens
+
+Tokens are predefined, contextual, single-character symbols, such as `@` and `$`, that have a special meaning to the C# compiler. C# supports the following tokens: 
```

Comment:
@rpetrusha I realized this morning that I wasn't much help by just saying that I wouldn't add a page for 'tokens'. 

The interesting problem is that neither symbol is an 'operator' or a 'keyword'. 
In the case of the `@` sign, it's a modifier on either a variable name or a string. In the case of `$`, it's modifying a string.

How about renaming the 'tokens' page as 'special characters'. Then, we can get this merged. In the meantime, I'll see if @madstorgersen has a better term.